game/theater/controlpoint.py

```diff
@@ -816,28 +853,26 @@ def frontline_unit_count_limit(self) -> int:
         return self.front_line_capacity_with(self.active_ammo_depots_count)
 
     @property
-    def all_ammo_depots(self) -> Iterator[BuildingGroundObject]:
+    def all_ammo_depots(self) -> Iterator[TheaterGroundObject]:
```

Comment:
i changed to tgo and removed the assert because it's not needed. The called function `is_ammo_depot` is a function of the TGO not the BuildingGroundObject. So we dont have to care about the actual subclass. I just cleaned it up